specificatie/openapi.yaml

```diff
@@ -107,7 +107,7 @@ paths:
                         Geeft aan dat de persoon een man of een vrouw is, of dat het geslacht (nog) onbekend is.
```

Comment:
in deze beschrijving moeten de mogelijke waarden er m.i. uit. dan blijft er wel weinig over dat iets toevoegt. @CathyDingemanse heb jij een voorstel voor een beschrijving van geslachtsaanduiding

Reply:
"Geeft aan wat het geslacht is van persoon." 

Dit voegt inderdaad weinig toe, dat ben ik met je eens @fsamwel

Reply:
Aangepast